polln_shift/views/shift.xml

```diff
@@ -124,6 +124,58 @@
         <xpath expr="//p[hasclass('h_irreg_autoextension')]" position="replace" />
     </template>
 
+    <template id="my_shift" name="Shifts for Irregular Workers"
+        inherit_id="beesdoo_website_shift.my_shift_next_shifts">
+        <xpath expr="//t[@t-esc='shift.task_type_id.name']" position="replace">
```

Comment:
Instead of replace, could you rather
- submit a PR to beesdoo_shift to replace `<t t-esc="shift.task_type_id.name" />` by `<t t-esc="shift.task_template_id.name or shift.task_type_id.name" />` 
- than adding the button with a `<xpath ... position="after">`